### ISSUE TITLE
'spaced-comment' in windows

### ISSUE THREAD

**Issue by smile-soul** [eslint/eslint#9143]
**Tell us about your environment**

* **ESLint Version: 4.2.0**
* **Node Version: 6.10.0**
* **npm Version: 3.10.0**

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint
**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

{
  "root": true,
  "parser": "babel-eslint",
  "env": {
    "browser": true,
    "node": true,
    "es6": true
    // "worker": true
  },
  "parserOptions": {
    "ecmaVersion": 6,
    "sourceType": "module",
    "ecmaFeatures": {
      "jsx": true
    }
  },
  "plugins": [
    "react"
  ],
  "extends": "airbnb",
  "rules": {
    "spaced-comment": 1,
    "react/jsx-no-bind": [
      "error",
      {
        "ignoreRefs": true,
        "allowArrowFunctions": false,
        "allowBind": true
      }
    ],
    "class-methods-use-this": "off",
    "react/prefer-stateless-function": ["off",
      {
        "ignorePureComponents": true
      }
    ],
    "import/no-extraneous-dependencies": "off",
    "no-else-return": "off",
    "linebreak-style": "off",
    "import/extensions": "off",
    "import/no-unresolved": "off",
    "react/prop-types": "off",
    "react/jsx-filename-extension": "off",
    "jsx-a11y/href-no-hash": "off",
    "react/require-default-props": "off",
    "no-console": "warn",
    "no-debugger": "off",
    "jsx-a11y/anchor-is-valid": [
      "warn",
      {
        "aspects": [
          "invalidHref"
        ]
      }
    ]
  },
  "globals": {
    "HAP": true
  }
}
```

</details>

**What did you do? Please include the actual source code causing the issue.**

```js
   /* user name*/
```

**What did you expect to happen?**
In windows,it will have a error.
**What actually happened? Please include the actual, raw output from ESLint.**
In windows, it dosen't refer to error, I can compile sucessefully. But In Unix it will have a error.

**Comment by eslintbot** [comment]
Hi @smile-soul, thanks for the issue. It looks like there's not enough information for us to know how to help you.

If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

Requesting a rule change? Please see [Proposing a Rule Change](http://eslint.org/docs/developer-guide/contributing/rule-changes) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!
